package.json

```diff
@@ -41,6 +41,7 @@
     "@changesets/read": "^0.5.3",
     "@manypkg/get-packages": "^1.1.3",
     "@octokit/plugin-throttling": "^5.2.1",
+    "@s0/ghcommit": "^1.2.1",
```

Comment:
I could swear I commented on this last time - but it seems I have not 😅 I think we need to inline this package here so we have full control over this code.

Reply:
Hmm, are you sure? It would involve quite a lot of code, and also include having to add things like codegen for GraphQL queries (or a reduction of Type Safety). There are also a bunch of integration tests that would significantly complicate the CI for this repo:

See:
- https://github.com/s0/ghcommit/blob/main/src/github/codegen.ts
- https://github.com/s0/ghcommit/blob/main/src/github/graphql/queries.ts
- https://github.com/s0/ghcommit/tree/main/src/test/integration

Seems like a lot of stuff to inline for something that has quite a nice abstraction boundary & test coverage.

I'm also pretty responsive to PRs and maintenance etc... but if you insist on ownership then perhaps we should consider transferring ownership of the repository & NPM package instead? (which I'm happy to do).

Reply:
Or perhaps you could also just fork this repo, and publish to a different NPM package too and use that?

Reply:
I'm usually very keen on using external packages - but this is a crucial piece of software that interacts with tokens and whatnot. So having a dependency on a package that is not well-established in the ecosystem makes me uneasy here.

I certainly don't really want to maintain it... 😅 but it feels like a necessity in this case. 

> perhaps we should consider transferring ownership of the repository & NPM package instead? (which I'm happy to do).

That sounds like the easiest option if you are willing to do that 👍

Reply:
Yeah happy to do that. Would you be happy for me to continue to maintain it under the changesets org?

Reply:
(Looks like I need to be added to the changesets org and have permission to create new repos (at least temporarily) to facilitate the transfer)

Reply:
> Would you be happy for me to continue to maintain it under the changesets org?

Yes. I'd like that 👍 

> (Looks like I need to be added to the changesets org and have permission to create new repos (at least temporarily) to facilitate the transfer)

Perhaps it would be slightly easier if you'd add me as admin to your repo so I could take care of the transfer? If that's too big of a hassle then I'll figure out how to invite you to the Changesets org with properly scoped permissions etc

Reply:
Invite sent: https://github.com/s0/ghcommit/invitations (you will need to accept before I can make you Admin I think)

Reply:
Accepted

Reply:
Hmm looks like I can only add you as a collaborator and can't make you an Admin (probably as it's a user repo not org repo) 🙃. So I think adding me to the org is the only way, can deffo be temporary though.

Reply:
Repo transferred, I'll work on re-publishing to new NPM package later on this week.

Reply:
Repo has been updated and ready to publish to `@changests/ghcommit`. Could you set up an `NPM_TOKEN` secret for the repo, so that CI can start publishing: https://github.com/changesets/ghcommit/actions/runs/14663639483/job/41153344520

You may need to manually build & publish an older version (i.e. the previous one) to be able to create a token with the appropriate scopes, and so that the build & publish workflow succeeds. (Definitely avoid publishing `main`/ the current version manually though, as it would mean provenance won't be included and we'll need to bump the version again)